src/ad-services/connatix/index.ts

```diff
@@ -0,0 +1,44 @@
+import { communicationService, eventsRepository, UapLoadStatus } from '@ad-engine/communication';
+import { BaseServiceSetup, context, utils } from '@ad-engine/core';
+import { ConnatixPlayer } from './connatix-player';
+import { initConnatixHeadScript } from './head-script';
+import { PlayerInjector, PlayerInjectorInterface } from './player-injector';
+
+export const logGroup = 'connatix';
+
+export class Connatix extends BaseServiceSetup {
+	get cid(): string {
+		return context.get('services.connatix.cid');
+	}
+
+	isEnabled(): boolean {
+		return context.get('services.connatix.enabled');
+	}
+
+	constructor(
+		protected instantConfig,
+		protected globalTimeout,
+		private playerInjector: PlayerInjectorInterface,
+	) {
+		super();
+
+		const cnxPlayer = new ConnatixPlayer();
```

Comment:
Could we please avoid using acronyms like this to prevent any confusion? 🙏🏻

```suggestion
		const connatixPlayer = new ConnatixPlayer();
```